task8_mafia_game/tests/test_menu.dart

```diff
@@ -0,0 +1,6 @@
+import '../entities/game_menu.dart';
+
+void main() async {
+  GameMenu menu = GameMenu();
+  await menu.start();

```

Comment:
this test need to test user inputs for menu
you need to write such code that can get some data and this data you need to mock